tests/playwright/assets/modules/favorites/widgets.js

```diff
@@ -0,0 +1,26 @@
+const { Notifications } = require( '../../../assets/assets/dev/js/editor/notifications' );
+
+exports.Widgets = class Widgets {
+	constructor( page ) {
+		this.page = page;
+	}
+
+	async add( widgetTitle ) {
+		const openPanelElementContextMenu = async () => {
+			await this.page.click( `.elementor-element >> :text("${ widgetTitle }")`, { button: 'right' } );
+			await this.page.waitForSelector( '.elementor-context-menu' );
+		};
+		await openPanelElementContextMenu();
+
+		if ( await this.page.$( ':text("Remove from Favorites")' ) ) {
+			await this.page.click( ':text("Remove from Favorites")' );
+			await openPanelElementContextMenu();
+		}
```

Comment:
Why do you test in a local environment?

Reply:
I can move to a testing environment, but it doesn't change the fact that previous test in the same run might try to add the same favorite. Therefore, it should be reset.

Reply:
The test should cleanup itself instead of clean up another test

Reply:
Done.